### ISSUE TITLE
Does nothing on Ubuntu 20.04

### ISSUE THREAD

**Issue by baldyman01** [Liemaeu/UE4-Manager-for-Linux#1]
I installed the .deb on Ubuntu 20.04. The Open the Marketplace button works. Download button just throws this -

/bin/sh: 1: cd: can't cd to /home/pete/UE4LinuxLauncher/Compiled/
/bin/sh: 1: ./start.sh: not found

**Comment by Liemaeu** [comment]
Sorry for the late response!

Did you use the Install option first?

Otherwise Erlandys Marketplace Downloader will not be installed.
You can do this manually:

```
cd ~/
git clone https://github.com/Erlandys/UE4LinuxLauncher.git
chmod +x UE4LinuxLauncher/Compiled/start.sh
```

I will change that in the future, so the Open the Marketplace button will do this automatically (if the Install button was never used).

**Comment by Liemaeu** [comment]
Fixed with [1.3 release](https://github.com/Liemaeu/UE4-Manager-for-Linux/releases/tag/1.3).
